cmd/apitest/http_server.go

```diff
@@ -86,12 +87,25 @@ func (ats *Suite) StopHttpServer() {
 }
 
 type ErrorResponse struct {
-	Error string `json:"error"`
+	Error string      `json:"error"`
+	Body  interface{} `json:"body,omitempty"`
 }
 
-func errorResponse(w http.ResponseWriter, statuscode int, err error) {
+func errorResponse(w http.ResponseWriter, statuscode int, err error, bodyBytes []byte) {
+	var body interface{}
+	if utf8.Valid(bodyBytes) {
+		if len(bodyBytes) > 0 {
+			body = string(bodyBytes)
+		} else {
+			body = nil
+		}
```

Comment:
Now we have to much magic inside errorResponse. The check for bodyBytes must not be in the scope of this function. Please roll these changes back and check "bodyBytes" for a different representation in /bounce-json.